src/peakrdl_markdown/exporter.py

```diff
@@ -194,9 +208,10 @@ def _add_addrmap_regfile(
         """
         members: List[MarkdownExporter.GenStageOutput] = []
         member_gen: str = ""
-        for child in node.children(unroll=True, skip_not_present=False):
-            if isinstance(child, (AddrmapNode, RegfileNode)):
-                output = self._add_addrmap_regfile(child, msg, depth - 1)
+        not_memory = not isinstance(node, MemNode)
```

Comment:
Please add a comment here about why we don't want to unroll MemNodes.

Reply:
Added comment about memories